TESTING.md

```diff
@@ -15,3 +17,33 @@ In powershell:
 ```
 
 Note: will need to press [Ctrl]+[C] after npm tests finish
+
+## Frontend tests
+
+The frontend ReactJS application currently has a test of unit tests using Enzyme.
+
+To run these tests, you'll need to enter the frontend Docker container.
+
+In powershell:
+
+```shell
+docker exec -it project-thundercat_frontend_1 /bin/bash
+
+npm run test
+```
+
+## Backend tests
+
+The backend Django application has a set of tests for it's views and uses Postman to manage API tests.
+
+To run the unit tests, you'll need to enter the frontend Docker container.
+
+In powershell:
+
+```shell
+docker exec -it project-thundercat_backend_1 /bin/bash
+
+./manage.py test
+```
+
+TODO(fnormand01) - Outline how to run Postman tests.
```

Comment:
@fnormand01 could you fill this in?

Reply:
@caleybrock I added some postman documentation.
See commit #https://github.com/code-for-canada/project-thundercat/pull/475/commits/fdef200a012447c8290b304a7775368faf3c88c7